dist/polkit-1/rules.d/zincati.rules

```diff
@@ -0,0 +1,8 @@
+// Allow Zincati to deploy, and finalize a staged deployment through rpm-ostree.
+polkit.addRule(function(action, subject) {
+    if ((action.id == "org.projectatomic.rpmostree1.deploy" ||
+         action.id == "org.projectatomic.rpmostree1.finalize-deployment") &&
+         subject.user == "zincati") {
```

Comment:
Minor/optional: the indentation here makes it look like that final conditional is part of the same nested parentheses. Maybe just de-indent by one space?

Reply:
Right, thanks for catching - fixed!